misc/python/materialize/mzcompose/__init__.py

```diff
@@ -29,7 +29,7 @@
 say = ui.speaker("C> ")
 
 
-DEFAULT_CONFLUENT_PLATFORM_VERSION = "7.5.0"
+DEFAULT_CONFLUENT_PLATFORM_VERSION = "7.5.2"
```

Comment:
No need to change it this time, but I don't like it that much to mix different changes in a PR (unless this is required for this PR).

Reply:
I should have given an explanation. I had to check if these versions exist on Dockerhub for arm64, and while doing that thought it makes sense to use the latest version anyway. But yes, strictly speaking doesn't belong into this PR.